cmd/ttn-lw-stack/commands/is_db.go

```diff
@@ -291,6 +292,56 @@ var (
 			return nil
 		},
 	}
+	isDBEUIBlockCreationCommand = &cobra.Command{
+		Use:   "create-eui-block",
+		Short: "Create an EUI block in IS db (currently only DevEUI block supported)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			logger.Info("Connecting to Identity Server database...")
+			db, err := gormstore.Open(ctx, config.IS.DatabaseURI)
+			if err != nil {
+				return err
+			}
+			defer db.Close()
+			if !config.IS.DevEUIBlock.Enabled {
+				logger.Error("DevEUIBlock configuration disabled")
+				return nil
+			}
```

Comment:
How is this relevant? Wouldn't it be possible to first create the block, and then later enable issuing from that block?